falcon/response.py

```diff
@@ -967,19 +1007,24 @@ def append_link(
         append_link
     )
 
-    cache_control = header_property(
+    cache_control: Union[str, Iterable[str], None] = header_property(
         'Cache-Control',
         """Set the Cache-Control header.
 
         Used to set a list of cache directives to use as the value of the
         Cache-Control header. The list will be joined with ", " to produce
         the value for the header.
-
         """,
         format_header_value_list,
     )
+    """Set the Cache-Control header.
```

Comment:
I don't like the duplication, but I'm not sure if there is another way of keeping the property doc set and have ide pick up the docs

Reply:
well let's keep for now. we can maybe improve on it later on